common/views/components/Buttons/LinkButton/README.md

```diff
@@ -0,0 +1,2 @@
+### Link Buttons
+These replicate the visual styles of 'Button Buttons', but use an `a` rather than a `button` element.
```

Comment:
As per above, do we actually use both?
I wonder if it's worth having 1 component if we do, that if you send the tag override, we return another component.

So a component of two components. From a design system POV it clears things up.

e.g.
```JS
export const ControlButton = ({ tag }) => (

)
const Button = () => (<button>🤡</button>)
const Link = () => (<a>🐝</a>)
```

Slightly more complicated in the component, but exposes a simpler component API.

Reply:
This way to - the exposed component would have only 1 set of props - this sounds good, but not sure if it's possible?